files/tasks.cron

```diff
@@ -0,0 +1,6 @@
+SHELL=/bin/sh
+RAILS_ENV=production
+FOREMAN_HOME=/usr/share/foreman
+
+# Clean up expired tasks from the database
+45 19 * * *     foreman    /usr/sbin/foreman-rake foreman_tasks:cleanup >>/var/log/foreman/cron.log 2>&1
```

Comment:
Usually we have line endings at the end of the file, this results in smaller diffs if you add another line.

Reply:
(done)